extension/threadpool/threadpool.cpp

```diff
@@ -93,25 +92,26 @@ void ThreadPool::run(
       0u);
 }
 
-// get_threadpool is not thread safe due to leak_corrupted_threadpool
-// Make this part threadsafe: TODO(kimishpatel)
```

Comment:
this comment is unrelated. I think (trying to remember) the reason why i mentioned this issue is because at fork we will hit `leak_corrupted_threadpool = false` at line 126 and that is being carried out in thread unsafe way. So after process fork if there are two threads call get_threadpool then `leak_corrupted_threadpool = false` can potentially be executed at the same time and can lead to unsafe behaviors